DRAFT.md

```diff
@@ -752,7 +751,7 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& a);
 7. _Mandates_: `T` is a complete type.
 
 8. _Effects_: `alloc` is direct-non-list-initialized with `a`. Value
-   initializes an owned object of type `T` using the specified allocator.
+   initializes an owned object of type `T`.
```

Comment:
I think this should be "constructs" not "value-initializes" because the latter prevents the allocator's construct member from providing additional args. Value-initialization means precisely `T()` and nothing else. Maybe "Constructs an owned object of type T with an empty argument list" ?

```suggestion
   Constructs an owned object of type `T` with an empty argument list.
```